Plugins/NativeInput/CMakeLists.txt

```diff
@@ -1,9 +1,20 @@
 set(SOURCES 
     "Include/Babylon/Plugins/NativeInput.h"
-    "Source/NativeInput.cpp"
-    "Source/NativeInput.h"
-    "Source/DeviceInputSystem.cpp"
-    "Source/DeviceInputSystem.h")
+    "Source/Shared/NativeInput.cpp"
+    "Source/Shared/NativeInput.h"
+    "Source/Shared/DeviceInputSystem.cpp"
+    "Source/Shared/DeviceInputSystem.h")
+
+if (ANDROID)
+    set(SOURCES ${SOURCES}
+        "Source/Android/NativeInputAndroid.cpp")
```

Comment:
Can we just name all these files `NativeInput.cpp`? I think that is more similar to how we do it in other places, like UrlLib.

Reply:
Can you include multiple cpp files with the same name in the same lib project?

Reply:
If they are in different folders I think so.

Reply:
yeah looks like this suggestion works. FWIW, I prefer this naming as well :)